src/main/java/graphql/ExecutionInput.java

```diff
@@ -48,6 +49,14 @@ private ExecutionInput(Builder builder) {
         this.cancelled = builder.cancelled;
     }
 
+    private static String assertQuery(Builder builder) {
+        if ((builder.query == null || builder.query.isEmpty()) && builder.extensions.containsKey("persistedQuery")) {
```

Comment:
We need a well known constant on ExecutionInput for this string value.  With Javadoc on it on how it needs to be set to allow PQs say